nb/documentation/creating_geometry/creating_geometry.md

```diff
@@ -74,57 +74,59 @@ Empty().view();
 ### Hexagon
 Creates a new hexagon
 
+![Image](creating_geometry.md.$11.png)
+
 ```JavaScript
 await Hexagon(4).view();
 ```
 
-![Image](creating_geometry.md.$11.png)
+![Image](creating_geometry.md.$12.png)
 
 ```JavaScript
 await Hexagon(6, 3, 2).view();
 ```
 
-![Image](creating_geometry.md.$12.png)
-
 ---
 ### Hershy
 A built in single line font useful for adding text
 
+![Image](creating_geometry.md.$14.png)
+
 ```JavaScript
 Hershey('Some Example Text', 20).by(align('xy')).view();
 ```
 
-![Image](creating_geometry.md.$14.png)
-
 ---
 ### Icosahedron
 Creates a new Icosahedron
 
+![Image](creating_geometry.md.$16.png)
+
 ```JavaScript
 await Icosahedron(10).view();
 ```
 
-![Image](creating_geometry.md.$16.png)
-
 ---
 ### Line
 Creates a new line
 
+![Image](creating_geometry.md.$18.png)
+
 ```JavaScript
 await Line(10).view();
 ```
 
-![Image](creating_geometry.md.$18.png)
-
 ---
 ### Link
 Links a set of ordered points together into ordered segments.
 
+![Image](creating_geometry.md.$20.png)
+
 ```JavaScript
 await Group(Point(0, 0), Point(10, 10), Point(20, 10)).link().view();
 ```
 
-![Image](creating_geometry.md.$20.png)
+![Image](creating_geometry.md.$21.png)
```

Comment:
These are all automatically numbered, right? You didn't have to change all of these?

Reply:
Yes :)